### ISSUE TITLE
Markdown Preview pane Scrolling Unhelpfully

### ISSUE THREAD

**Issue by Obscure2020** [mjbvz/vscode-github-markdown-preview-style#85]
While writing a markdown file about an esolang I'm working on, I encountered some issues with your extension related to scrolling. Here is a recording of the issues I've encountered:
[![vscode001-thumbnail](https://yt-embed.herokuapp.com/embed?v=0Eu_v87gQAQ)](https://youtu.be/0Eu_v87gQAQ "vscode001")

At first, I thought these were VSCode issues. Then, when I began writing a bug report, I came across Microsoft's suggestion to try again with all the extensions deactivated. Sure enough, these issues disappeared. Since this is the only Markdown-related extension I have installed, it must be an issue with this extension.

I hope you can figure out what's wrong with the scrolling! I really do appreciate your extension. Github is the only site I ever need to write markdown for, so being able to see Github-flavored-and-styled output while I write in VSCode is extremely useful.

**Comment by Obscure2020** [comment]
On further investigation, I think this might actually be a VSCode problem rather than a problem with this extension. I came to this conclusion after realizing how simple this extension really is, and also experiencing the exact same issues with a different, identical extension.

I have opened a VSCode issue on this topic, and I will update/close this issue here if anything happens over there.
[https://github.com/microsoft/vscode/issues/143984](https://github.com/microsoft/vscode/issues/143984)
